docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,231 @@
+[role="xpack"]
+[[anomaly-detection-scale]]
+= Using {anomaly-detect} at scale
+
+Working with large {anomaly-jobs} requires careful planning; otherwise, job 
+performance could be adversely effected. This page contains several guidelines 
+to consider when using {anomaly-detect} at scale and helps you to avoid possible 
+pitfalls.
```

Comment:
`This page contains several guidelines and considerations when using {anomaly-detect} at scale.`

The previous intro sounds negative. It is not about pitfalls and adverse affects. Jobs at scale require resources at scale, and we're providing some guidelines and things to think about to get the best out of anomaly detection.